src/doc/palette.cpp

```diff
@@ -400,6 +408,117 @@ int Palette::findBestfit2(int r, int g, int b, int a) const
   return bestfit;
 }
 
+// Auxiliary function for rgbToOtherSpace()
+double f(double t)
+{
+  if (t > 0.00885645171)
+    return pow(t, 0.3333333333333333);
+  else
+    return (t / 0.12841855 + 0.137931034);
+}
+
+// Auxiliary function for findBestfit3()
+void rgbToOtherSpace(double& r, double& g, double& b, FitCriteria fc)
+{
+  if (fc == RGB)
+    return;
+  double Rl, Gl, Bl;
+  // Linearization:
+  r = r / 255.0;
+  g = g / 255.0;
+  b = b / 255.0;
+  if (r <= 0.04045)
+    Rl = r / 12.92;
+  else
+    Rl = pow((r + 0.055) / 1.055, 2.4);
+  if (g <= 0.04045)
+    Gl = g / 12.92;
+  else
+    Gl = pow((g + 0.055) / 1.055, 2.4);
+  if (b <= 0.04045)
+    Bl = b / 12.92;
+  else
+    Bl = pow((b + 0.055) / 1.055, 2.4);
+  if (fc == linearizedRGB) {
```

Comment:
I think we can use a switch-case here:
```suggestion
  switch (fc) {
    case FitCriteria::linearizedRGB:
      ...
      break;
  }  
```